bin/node/runtime/src/lib.rs

```diff
@@ -573,6 +573,7 @@ parameter_types! {
 	pub const SignedRewardBase: Balance = 1 * DOLLARS;
 	pub const SignedDepositBase: Balance = 1 * DOLLARS;
 	pub const SignedDepositByte: Balance = 1 * CENTS;
+	pub const ChallengeDepositDiff: Balance = 1 * DOLLARS;
```

Comment:
As noted before, please make this a ratio: The challenger would gain a ratio of the chalengee's deposit as reward.

Reply:
will update this